package.json

```diff
@@ -8,7 +8,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "nodemon",
+    "start": "node ./src/server/server.js",
```

Comment:
Tried running `npm start` and it threw:

100x-backend@1.0.0 start /Users/jayhwang/.sandbox/projects/todo/100x/100x-Backend
node ./src/server/server.js

module.js:544
    throw err;
    ^

Error: Cannot find module 'dotenv'
    at Function.Module._resolveFilename (module.js:542:15)
    at Function.Module._load (module.js:472:25)
    at Module.require (module.js:585:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/jayhwang/.sandbox/projects/todo/100x/100x-Backend/src/server/server.js:1:63)
    at Module._compile (module.js:641:30)
    at Object.Module._extensions..js (module.js:652:10)
    at Module.load (module.js:560:32)
    at tryModuleLoad (module.js:503:12)
    at Function.Module._load (module.js:495:3)